staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -389,7 +407,12 @@ func NewConfig(codecs serializer.CodecFactory) *Config {
 		}
 
 		hash := sha256.Sum256(hashData)
-		id = "apiserver-" + strings.ToLower(base32.StdEncoding.WithPadding(base32.NoPadding).EncodeToString(hash[:16]))
```

Comment:
this is a breaking change, are we sure we want to do ti here?

Reply:
created https://github.com/kubernetes/kubernetes/pull/118958 for this, but I need this change in this PR in order for the integration test to work.